apps/src/applab/appStorage.js

```diff
@@ -2,13 +2,62 @@
 
 /* global Applab */
 
-var RecordListener = require('./RecordListener');
+var Firebase = require("firebase");
 
 /**
  * Namespace for app storage.
  */
 var AppStorage = module.exports;
 
+// HACKHACK: Need to make this configurable
+var BASE_DB_URL = 'https://radiant-fire-5518.firebaseio.com';
```

Comment:
Is this URL is the equivalent of an API key? This raises the question of how we're going to protect our API keys, which if stolen might allow unlimited access to our firebase account at our expense.